language-server/modules/langserver-core/src/test/resources/completion/statement_context/source/assignment_stmt_ctx_source10.bal

```diff
@@ -0,0 +1,20 @@
+type MyType record {|
+    int var1;
+    string var2;
+|};
+
+function getString(string myStr) returns string {
+    return "hello";
+}
+
+function getInt() returns int {
+    return 1;
+}
+
+function myFunction(function () returns string func,string paramStr, MyType myType) {
+    int myInt = 10;
+    string myStr = "myStr";
+    string hello;
+
+    hello = m
+}
```

Comment:
Let's add another test for assigning to a variable of a user defined type as well. Example:

```
MyType t = <cursor>;
```

Reply:
done!